techMail.js

```diff
@@ -34,11 +34,11 @@ module.exports = function(configuration, templatesDir) {
 
             emailTemplates(templatesDir, function(err, template) {
                 if (err) {
-                    logger.error(err);
+                    logger.error("Unable to configure the template engine... - Details:", err && err.stack || err);
```

Comment:
I think that the `{}` message was logged in the last step `transporter.sendMail(...` because the list of recipients was empty. But yes, it means that in some case, the `err` is an empty object, so I am just trying to add more context to better identify further issues we could encounter.